packages/InputGroup/style.js

```diff
@@ -0,0 +1,69 @@
+import styled from 'styled-components'
+
+export const LabelStyle = styled.label(() => ({
+  span: {
+    fontSize: '14px',
+    color: '#2A2C2E',
+    display: 'block',
+  },
+}))
+
+export const InputGroupStyle = styled.div(({ hasValue }) => ({
+  display: 'inline-flex',
+  flexWrap: 'nowrap',
+  height: '52px',
```

Comment:
Is this a new design direction? Acceptance criteria in #228 states that the input field should be 40px high, also that the button should be 40px x 40px

Reply:
Yes, that is based on design. You can talk to Laurel for more details.

Reply:
ok, thanks